modules/nw-route-admission-policy.adoc

```diff
@@ -34,3 +34,20 @@ spec:
     namespaceOwnership: InterNamespaceAllowed
 ...
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to configure the route admission policy:
+[source,yaml]
+----
+apiVersion: v1
+kind: IngressController
+metadata:
+  name:default
+  namespace:openshift-ingress-operator
+spec:
+  replicas: 2
+  routeAdmission:
+    namespaceOwnership: InterNamespaceAllowed
```

Comment:
Need to fix the API group name and add spaces after `name:` and `namespace:`, and presumably setting `replicas: 2` isn't supposed to be part of the pocedure, so it should be deleted:
```suggestion
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  name: default
  namespace: openshift-ingress-operator
spec:
  routeAdmission:
    namespaceOwnership: InterNamespaceAllowed
```